packages/Dialog/package.json

```diff
@@ -26,8 +26,8 @@
   "dependencies": {
     "@igloo-ui/tokens": "^1.0.0",
     "classnames": "^2.3.1",
-    "@igloo-ui/modal": "^1.0.0",
-    "@igloo-ui/button": "^0.3.0"
+    "@igloo-ui/modal": "*",
+    "@igloo-ui/button": "*"
```

Comment:
just wondering why the versions were removed here?

Reply:
in fact this is the method to use with Lerna to link packages together. By doing this it is able to see if a version change during the release should be reflected on the linked packages. 

https://lerna.js.org/docs/getting-started#bootstrapping-projects